internal/pkg/cli/env_init.go

```diff
@@ -569,6 +570,7 @@ func (o *initEnvOpts) deployEnv(app *config.Application, customResourcesURLs map
 		CustomResourcesURLs:      customResourcesURLs,
 		AdjustVPCConfig:          o.adjustVPCConfig(),
 		ImportVPCConfig:          o.importVPCConfig(),
+		ImportCertARNs:           o.importCerts,
```

Comment:
Can we capture the imported certs in SSM as well? I believe we need to put it in SSM so that `env upgrade` can work and generated the template with imported certs.   
Would you mind testing for that so that we ensure environments with imported VPCs/certs can be upgraded

Reply:
OK done updating!